### PR TITLE
Update readme tutorial

### PR DESCRIPTION
**Pull request by kt-9**
Added material

### CODE REVIEW

**Comment by kt-9** [comment]
@lina2000

**Comment by kt-9** [comment]
@lina2000 See how it looks here https://github.com/lina2000/D1/tree/update_readme_tutorial

File: README.md [hunk 1]
```diff
@@ -12,3 +12,153 @@ My solutions to problems I solve on https://www.hackerrank.com/contests/d1/chall
...
+```
+This will print `Go to school!`.
+
+---
+
+```
+int temperature = 0;
+if (temparature > 0) {
+  printf("Snow is melting!");
+} else if (temperature < 0) {
```
Comment (lina2000) [inline README.md]: Can I do this? I mean print 'else if'. So, in my mind, to add the other 'else' (below), you have to write 'if' after else. Am I right?

File: README.md [hunk 2]
```diff
@@ -12,3 +12,153 @@ My solutions to problems I solve on https://www.hackerrank.com/contests/d1/chall
...
+  digit = digit + 1;
+}
+```
+
+---
+
+Read numbers from standard input until the user enters `0`:
+```
+int entered_number = -1; // Nothing is entered yet.
+while (entered_number != 0) {
```
Comment (lina2000) [inline README.md]: This means not equal, don't it?

File: README.md (hunk 1 above)
Comment (kt-9) [reply README.md]: Yes. This is useful when you need multiple if conditions. This is equivalent to
```
if (temparature > 0) {
  printf("Snow is melting!");
} else {
  if (temperature < 0) {
    printf("Water is freezing");
  } else {
    printf("Water and ice");
  }
}
```

File: README.md (hunk 2 above)
Comment (kt-9) [reply README.md]: Yes.
`==` means equal
`!=` means not equal

**Pull request merged**